GVRf/Framework/framework/src/main/java/org/gearvrf/GVRContext.java

```diff
@@ -193,6 +194,19 @@ public GVRActivity getActivity() {
     }
 
     /**
+     * Get the {@link GVRImporter} to use for loading assets.
+     * 
+     * A {@code GVRImporter} loads models asynchronously from your applications's
+     * local storage or the network.
+     * 
+     * 
+     * @return The asset loader associated with this context.
+     */
+    public GVRImporter getAssetLoader() {
```

Comment:
The function description says that the call would return an asset loader but it returns an importer. We would decide on a name either rename the call to getImporter or rename GVRImporter to AssetLoader.